bullet_train-incoming_webhooks/config/models/roles.yml

```diff
@@ -0,0 +1,40 @@
+# This file is just here to make tests pass when run from inside this gem directory.
+# Seems like maybe we need a better resolution method for this file?
+# It wasn't found if I put this file in test/dummy/config/models/
```

Comment:
It looks like we just use `./config/models/roles.yml` to find the file. Maybe we can use the `Dir.glob` or something similar to find the file more dynamically and prioritize the `test/dummy` directory when working in the test environment.

https://github.com/bullet-train-co/bullet_train-core/blob/445b8e616199782d4de2de96b1e4e319656b08ff/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb#L486-L487

Reply:
Ah, I'd bet we want:

```ruby
role_file = File.join(Rails.root, "config", "models", "roles.yml")
```

Reply:
Aaaaand, it's not just there that we need to fix. These two lines also point to that file directly, and just throwing a `Rails.root` in there doesn't seem to work. I'm just going to leave all of that stuff alone and will open an issue about resolving those paths differently.